Misc/NEWS

```diff
@@ -984,6 +984,15 @@ C API
 Documentation
 -------------
 
+- Issue #30052: the link targets for ``:func:\`bytes\``` and
```

Comment:
Hmm, I can't work out how to get `make suspicious` to be OK with the escaped markup, so I've asked python-dev for advice: https://mail.python.org/pipermail/python-dev/2017-May/147820.html